src/utils.rs

```diff
@@ -83,6 +85,17 @@ pub fn rewrite_snapshot(target_snapshot: &mut [SnapshotPath]) {
     }
 }
 
+pub fn fn_regex_rewrite(
+    pattern: String,
+    rewrite: String,
+) -> Box<dyn Fn(String) -> Result<String> + Sync + Send> {
+    Box::new(move |data| {
+        let re = regex::Regex::new(pattern.as_str())
```

Comment:
Create regex every time we call this function may lead to a huge overhead. Consider create this regex upon loading config.